_posts/2017/2017-03-28-epages-hackathon-2017.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: "ePages Hackathon 2017"
+date: "2017-03-28 06:39:20"
```

Comment:
Change date to 29th of March (also in file)
I had to reschedule Benjamins post, sorry.